### PR TITLE
.bib-file corrections

### PR DESCRIPTION
**Pull request by QuLogic**
https: and consistency fixes. Also cleaned up a few things that shouldn't be there. See commit messages for specific changes.

### CODE REVIEW

File: paper.bib [hunk 1]
```diff
@@ -59,7 +59,7 @@ @article{strobel_when_2009
 }
 
 @article{markham_project_2011,
-	title = {Project {Based} {Learning} {A} {Bridge} {Just} {Far} {Enough}},
```
Comment (joelostblom) [inline paper.bib:64]: @QuLogic Any chance you know how I can control this in Zotero? The line above is what is exported when I chose `BibTex`. The only other sensible choice is `BibLatex`, but that creates `title = {Project Based Learning A Bridge Just Far Enough},`, which still has the capitals. Not sure what the presence of the additional `{}` around words is for either.

Comment (joelostblom) [inline paper.bib:64]: @QuLogic Any chance you know how I can control this in Zotero? The line above is what is exported when I chose `BibTex`. The only other sensible choice is `BibLatex`, but that creates `title = {Project Based Learning A Bridge Just Far Enough},`, which still has the capitals. Not sure what the presence of the additional `{}` around words is for either.

**Review by joelostblom — APPROVED** [review]

**Comment by joelostblom** [comment]
Thanks @QuLogic!

File: paper.bib (hunk 1 above)
Comment (QuLogic) [reply paper.bib:64]: Well, I use the Better BiBTeX extension, but I'm not sure that does anything. I just remove all the unnecessary capitals from the titles. The `{}` forces letter case to be immutable, which is wrong here (though fine in other entries where I left it.) It should be up to the bibliography style.

**Pull request merged**